src/pages/index.js

```diff
@@ -226,4 +319,21 @@ closeModalButtons.forEach((button) => {
   button.addEventListener("click", () => closeModal(modal));
 });
 
+//performs all network fetchs and handles the responses
+//getAppInfo returns Promise.all.
+api
+  .getAppInfo()
+  .then(([cards, userInfo]) => {
```

Comment:
Good job that you get the whole info with `Promise.all`

Reply:
Thanks for all of your feedback!